core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPools.java

```diff
@@ -104,81 +104,86 @@ public static void resizePool(final ThreadPoolExecutor pool, final AccumuloConfi
    */
   @SuppressWarnings("deprecation")
   public static ExecutorService createExecutorService(final AccumuloConfiguration conf,
-      final Property p) {
+      final Property p, boolean emitThreadPoolMetrics) {
```

Comment:
Could add some javadoc for this param like the following

```java
@param emitThreadPoolMetrics When set to true will emit metrics and register the metrics in a static registry.  After the thread pool is deleted, there will still be metrics objects related to it in the static registry.  There is no way to clean these left over objects up therefore its recommened that this option only be set true for long lived thread pools.  Creating lots of short lived thread pools and registering them can lead to out of memory errors over long time periods.  
```

Reply:
Addressed in 1a7f098